tests/basic/storage_pool_test.go

```diff
@@ -9761,3 +9761,190 @@ var _ = Describe("{AddDriveMetadataPool}", func() {
+				expectedSize := (selectedPool.TotalSize / units.GiB) + driveSize
+				err = Inst().V.ExpandPool(selectedPool.Uuid, api.SdkStoragePool_RESIZE_TYPE_ADD_DISK, expectedSize, false)
+				log.FailOnError(err, "error while expanding pool")
+				isjournal, err := isJournalEnabled()
+				log.FailOnError(err, "Failed to check is journal enabled")
+				resizeErr := waitForPoolToBeResized(expectedSize, selectedPool.Uuid, isjournal)
+				if i == maxDrivesAllowed+1 {
+					poolStatus, err := getPoolLastOperation(selectedPool.Uuid)
+					log.FailOnError(err, "error getting pool status")
+					dash.VerifyFatal(resizeErr != nil, true, fmt.Sprintf("Expected new size to be '%d' or '%d'", expectedSize, expectedSize-3))
+					if poolStatus != nil {
+						dash.VerifyFatal(strings.Contains(poolStatus.Msg, "unable to add 1 new drive(s) as it would exceed maximum supported drives (6)"), true, "Error expected as drive added more than allowed per pool")
+					} else {
+						err := errors.New("failed while getting msg")
+						log.FailOnError(err, "failed while getting msg")
+					}
+				} else {
+					dash.VerifyFatal(resizeErr != nil, false, fmt.Sprintf("Expected new size to be '%d' or '%d'", expectedSize, expectedSize-3))
+				}
+			}
+			err = Inst().V.RefreshDriverEndpoints()
+			log.FailOnError(err, "error refreshing volume endpoints")
+		})
+
+		Step("Test max pool count", func() {
+			log.InfoD(stepLog)
+			poolList, err := GetPoolsDetailsOnNode(selectedNode)
+			poolLeft := maxPoolLength - len(poolList)
+			err = addNewPools(selectedNode, poolLeft)
+			log.FailOnError(err, "error adding new pools")
+			err = addNewPools(selectedNode, 1)
+			dash.VerifyFatal(err != nil, true, "Error expected as pools trying to create is more than allowed")
+			poolList, err = GetPoolsDetailsOnNode(selectedNode)
+		})
+
+		Step("Test max drives per node", func() {
+			log.InfoD(stepLog)
+			var selectedPools string
+			poolList, err := GetPoolsDetailsOnNode(selectedNode)
+			log.FailOnError(err, "fialed while getting updated node pool list")
+			//first add all the new pools
+			poolLeft := maxPoolLength - len(poolList)
+			err = addNewPools(selectedNode, poolLeft)
+			drvNum, err := Inst().N.RunCommand(selectedNode, "lsblk -l -d -e 11 -n -o NAME|wc -l", node.ConnectionOpts{
+				Timeout:         defaultCommandTimeout,
+				TimeBeforeRetry: defaultCommandRetry,
+				IgnoreError:     false,
+				Sudo:            false,
+			})
+			drvNum1 := strings.TrimSpace(string(drvNum))
+			drvNum1 = strings.Trim(drvNum1, "\n")
+			drvNumInt, err := strconv.Atoi(drvNum1)
+			log.FailOnError(err, "fialed to convert to int")
+			maxDriveLimit, err := GetPoolMaxCloudDriveLimit(&selectedNode)
+			maxDriveLimitInt := int(maxDriveLimit)
+			maxDriveAllowed := maxDriveLimitInt - drvNumInt
+			err = Inst().V.RefreshDriverEndpoints()
+			log.FailOnError(err, "error refreshing volume endpoints")
+			//expand by adding disk
+			selectedNode1, err := node.GetNodeByName(selectedNode.Name)
```

Comment:
can you please rename this variable instead of selectedNode1

Reply:
Done